src/components/data-grid/DataGrid.tsx

```diff
@@ -82,6 +82,7 @@ const DEFAULT_ROW_MODEL_TYPE = RowModelType.serverSide;
 const DEFAULT_HEIGHT = 'calc(100vh - 200px)';
 const DEFAULT_ROW_HEIGHT = 40;
 const PINNED_COLUMN_WIDTH = 54;
+const NO_TEAM = 'noTeam';
```

Comment:
`NO_TEAM` is only for this one particular use case when it is being used in `HRM/users`. Isn't there a way to make this generic? 

For example by adding an `emptyGroup` property to the `column` ? Because this way it cannot be reused anywhere else